core/Gunns.cpp

```diff
@@ -46,6 +46,7 @@ LIBRARY DEPENDENCY:
 #include <cmath>
```

Comment:
I don't see you using a size_t anywhere, is this still needed?  If not, please delete & resubmit.

Reply:
Reverted.